src/dust/dust.cpp

```diff
@@ -86,6 +88,10 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   const Real cfl_number = pin->GetOrAddReal("dust", "cfl", 0.8);
   params.Add("cfl", cfl_number);
 
+  // possible user_dt
+  const Real user_dt = pin->GetOrAddReal("dust", "user_dt", 1.0e10);
+  params.Add("user_dt", user_dt);
+
```

Comment:
Parthenon provides a `dt_user` now, so you can remove this

Reply:
I will remove that